src/Resources/config/routing.yml

```diff
@@ -13,11 +13,21 @@ bitbag_sylius_wishlist_plugin_shop_wishlist_remove_product:
     defaults:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.remove_product_from_wishlist
 
+bitbag_sylius_wishlist_plugin_shop_wishlist_remove_selected_products:
+    path: /wishlist/selected/remove
```

Comment:
Why don't follow the REST scheme? `/wishlist/{id}/products` and distinguish via HTTP methods?

Reply:
I can't send selected products id's in wildcard

Reply:
So `DELETE /wishlist` and query string argument as array will be definitely more readable.